subprojects/gradle/impact-shared-test-fixtures/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id("kotlin")
+    `maven-publish`
+    id("com.jfrog.bintray")
+}
+
+dependencies {
+    implementation(gradleApi())
+    implementation(project(":gradle:impact-shared"))
+    implementation(project(":gradle:test-project"))
+    implementation(Dependencies.test.truth)
+}
```

Comment:
These fixtures are reused between ImpactAnalysis plugin and CD plugin in integration tests.